lib/Gateway.js

```diff
@@ -410,7 +411,18 @@ function onNodeStatus (node) {
       data = { time: Date.now(), value: node.ready, status: node.status }
     }
 
-    this.mqtt.publish(topic, data)
+    this.mqtt.publish(nodeTopic + '/status', data)
+  }
+
+  // Publish Node Info on separate topic
+  // remove bulky  data like hassDevices, Groups and values
+  if (this.config.publishNodeDetails) {
```

Comment:
Shouldn't this be done only once and when node is ready ?

Reply:
We have different updates there, also neighbs, which can change! this is called when the node status changes. I think is good to have it done more often. Anothe rsolution would be to merge it with status topic in this case.

Reply:
@aretakisv Ok but if the node is not ready you could get some undefined here

Reply:
if node is not ready, this will return the node info from cache.
Also node.ready is exposed on payload! which can help people identify if ready or not!

here is an example of unready node
node.status and node.ready and node.isBeaming are self explanatory

```
{
  "id": 97,
  "deviceId": "",
  "manufacturer": "",
  "manufacturerId": "",
  "productType": "",
  "productId": "",
  "name": "Sensor",
  "loc": "Hallway",
  "neighbors": [],
  "ready": false,
  "available": false,
  "failed": false,
  "lastActive": null,
  "interviewCompleted": false,
  "firmwareVersion": "",
  "isBeaming": false,
  "isSecure": false,
  "keepAwake": false,
  "maxBaudRate": null,
  "isRouting": null,
  "isFrequentListening": false,
  "isListening": false,
  "status": "Unknown",
  "interviewStage": "Complete"
}
```

Reply:
ok no problem for me :)

Reply:
remember to fix docs

Reply:
I will! i will add these info too! for users